### PR TITLE
changes bolt-action recoil to be more in-line with other .30 rifles

### PR DESCRIPTION
**Pull request by Hatterhat**
## About The Pull Request
changes recoil around for the bolt-action guns:
excel mosin recoil_buildup from **40!!** -> 1.6
novakovic recoil buildup from 0.4 -> 1.7
handmade joonk boltaction from **40!!** -> 1.9

## Why It's Good For The Game
two bolt-actions with silly recoil buildups is a silly concept.

recoil is a scuffed system but these numbers really don't matter unless you reduce the fire delay on these bitches with the dangerzone/forged barrel

## Changelog
:cl:
balance: bolt-actions' recoil has been made a little more in-line with other .30 firing rifles
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/projectile/boltgun.dm [hunk 1]
```diff
@@ -100,7 +100,7 @@
 	icon_state = "boltgun_wood"
 	item_suffix  = "_wood"
 	force = 23
-	recoil_buildup = 0.4 // Double the excel variant
+	recoil_buildup = 4 // higher than excel variant though not too much?
```
Comment (Jackofashepard) [inline code/modules/projectiles/guns/projectile/boltgun.dm:103]: this means shot 2 will 50/50 go wide

File: code/modules/projectiles/guns/projectile/boltgun.dm [hunk 2]
```diff
@@ -118,7 +118,7 @@
 	fire_delay = 17 // abit more than the serbian one
 	damage_multiplier = 1
 	penetration_multiplier = 1
-	recoil_buildup = 40 //same as AMR
+	recoil_buildup = 5 // joonk gun probably doesn't handle recoil very well
```
Comment (Jackofashepard) [inline code/modules/projectiles/guns/projectile/boltgun.dm:121]: shot 2 will 90% go wide

File: code/modules/projectiles/guns/projectile/boltgun.dm [hunk 3]
```diff
@@ -14,7 +14,7 @@
 	fire_delay = 12 // double the standart
 	damage_multiplier = 1.4
 	penetration_multiplier = 1.5
-	recoil_buildup = 40 //same as AMR
+	recoil_buildup = 2
```
Comment (Jackofashepard) [inline code/modules/projectiles/guns/projectile/boltgun.dm:17]: 3 shots no problem

**Review by Jackofashepard — COMMENTED** [review]
The below only applies if you bolt faster than the recoil reset.

**Comment by Hatterhat** [comment]
i'm gonna be real with you man i have no idea how any of these numbers actually feel

**Comment by mattroks101** [comment]
> i'm gonna be real with you man i have no idea how any of these numbers actually feel

Test locally and find out then.

**Comment by Hatterhat** [comment]
good point, will do eventually (tm)

**Comment by Hatterhat** [comment]
i did actually test this now, in case the title change wasn't clear

**Review by Firefox13 — APPROVED** [review]

**Pull request merged**